community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/PropertyRecord.java

```diff
@@ -61,10 +64,33 @@ public PropertyRecord( long id )
     public PropertyRecord( long id, PrimitiveRecord primitive )
     {
         super( id );
-        setCreated();
         primitive.setIdTo( this );
     }
 
+    public PropertyRecord initialize( boolean inUse, PrimitiveRecord primitive, long prevProp, long nextProp )
+    {
+        super.initialize( inUse );
+        setCreated(); // TODO why?
```

Comment:
It has always done that. It stands out from the rest of the code. Someone should figure out at some point. This is no change in behaviour.